modules/apps/headless-apio/forms/forms-apio-client-test/src/testIntegration/java/com/liferay/forms/apio/client/test/internal/activator/SelectFieldFormApioTestBundleActivator.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.forms.apio.client.test.internal.activator;
+
+import com.liferay.dynamic.data.mapping.annotations.DDMForm;
+import com.liferay.dynamic.data.mapping.annotations.DDMFormField;
+
+/**
+ * @author Marcelo Mello
+ */
+public class SelectFieldFormApioTestBundleActivator
```

Comment:
Rename to SelectFormApioTestBundleActivator

Reply:
Renamed was requested. Thanks.